### ISSUE TITLE
Introduction to Data Management Plans

### ISSUE THREAD

**Issue by jwikle** [imls-dmt/resources-workflow#3]
Information about registered data management training resource

- [ ] Date resource added: 08/27/2018
- [ ] Registered by: jwikle
- [ ] Clearinghouse URL: http://dmtclearinghouse.esipfed.org/node/10180

**Comment by njhoebel** [comment]
Made the following changes to this record:
1.  Added more descriptive information about this video to highlight the professor giving the presentation as that is one of the most unique aspects of this particular resource.  There are a lot of resources that cover DMPs fairly generically, but the fact that this one is offered by a senior researcher makes it more useful.  I also added more information that put this resource in context of the full Data Management Plan Tutorial that covers the whole topic more completely.  Finally, added the URL to the video itself on YouTube for more direct access if people desired.
2.  Corrected the author names and affiliation.
3.  Corrected the license information (including adding it to the controlled value list as it wasn't listed on the choices!).
4.  Changed the publisher to the Penn State Department (not sure where the Northwestern info came from as I couldn't find a reference to Northwestern at all).
5.  Added mid and early career scientists to the Target Audience as well as Research faculty since this is coming from an academic environment.
6.  Changed the time to complete to be more than an hour, less than a day as the video was 1:51 from the video replay.  
7.  Have moved to publish & published on DMTC.
